openquakeplatform/bin/deploy.sh

```diff
@@ -0,0 +1,303 @@
+    
+    # enable wsgi apache
+    sudo apt-get install libapache2-mod-wsgi
+    sudo a2enmod wsgi
+    sudo invoke-rc.d apache2 restart
+    
+    # Create local_settings with pavement from repo
+    paver -f $HOME/$GIT_REPO/pavement.py oqsetup -l $LXC_IP -u localhost:8800 -s $HOME/geonode/data -d geonode -p $gem_db_pass -x $LXC_IP -g localhost:8080 -k $SECRET
+    sudo mv /etc/geonode/local_settings.py /etc/geonode/geonode_local_settings.py                                                                                                                                    
+    sudo cp  $HOME/$GIT_REPO/local_settings.py /etc/geonode/
+    sudo ./package/oq_install.sh -s post $HOME/$GIT_REPO/openquakeplatform/common/geonode_install.sh
+    sudo ./package/oq_install.sh -s setup_geoserver $HOME/$GIT_REPO/openquakeplatform/common/geonode_install.sh
+    
+    sudo sed -i '1 s@^@WSGIPythonHome '"$HOME"'/env\n@g' /etc/apache2/sites-enabled/geonode.conf
+    sudo invoke-rc.d apache2 restart                      
+}
+
+function apply_data() {
+    cd $HOME
+    geonode syncdb  
+    geonode migrate
+    geonode vuln_groups_create
+    geonode add_user $HOME/oq-private/old_platform_documents/json/auth_user.json
+    geonode loaddata $HOME/$GIT_REPO/openquakeplatform/dump/base_topiccategory.json
+    geonode import_vuln_geo_applicability_csv $HOME/$GIT_REPO/openquakeplatform/vulnerability/dev_data/vuln_geo_applicability_data.csv
+    geonode loaddata $HOME/$GIT_REPO/openquakeplatform/vulnerability/post_fixtures/initial_data.json
+    geonode loaddata -v 3 --app vulnerability $HOME/oq-private/old_platform_documents/json/all_vulnerability.json
+    geonode create_gem_user
+    pip install simplejson==2.0.9
+    # export CATALINA_OPTS="-Xms4096M -Xmx4096M"
+    sudo sed -i 's/-Xmx128m/-Xmx4096m/g' /etc/default/tomcat7
+    sudo service tomcat7 restart
+    
+    cd $HOME/$GIT_REPO
+    $HOME/$GIT_REPO/openquakeplatform/bin/oq-gs-builder.sh populate -a $HOME/oq-private/old_platform_documents/output "openquakeplatform/" "openquakeplatform/" "openquakeplatform/bin" "oqplatform" "oqplatform" "geonode" "geonode" "$gem_db_pass" "/var/lib/tomcat7/webapps/geoserver/data"
+    
+    cd ~/
+    # Put sql for all layers
+    for lay in $(cat $HOME/oq-private/old_platform_documents/sql_layers/in/layers_list.txt); do
+        sudo -u postgres psql -d $GEO_DBUSER -c '\copy '$lay' FROM '$HOME/oq-private/old_platform_documents/sql_layers/out/$lay''
+    done
+    
+    sudo cp -r $HOME/oq-private/old_platform_documents/thumbs/ $HOME/env/lib/python2.7/site-packages/geonode/uploaded/
+    sudo chmod 777 -R $HOME/env/lib/python2.7/site-packages/geonode/uploaded/thumbs
+    sudo cp -r $HOME/$GIT_REPO/openquakeplatform/common/gs_data/documents $HOME/env/lib/python2.7/site-packages/geonode/uploaded/
+    geonode add_documents_prod
+    # geonode updatelayers
+    # geonode sync_geofence
+
+    sudo invoke-rc.d apache2 restart
```

Comment:
I would replace `invoke-rc.d` (here and everywhere it is used) with `service`. `invoke-rc.d` is a Systm V utility and is deprecated.

And you are already using `service` (see below), so I would not mix different commands for the same operation